### ISSUE TITLE
How to evaluate on ImageNet?

### ISSUE THREAD

**Issue by JohnDreamer** [CompVis/taming-transformers#155]
Hi, it's a great work！And thanks for releasing the code! But I have a question --how to evaluate on ImageNet? In other words, should I get the FID scores on the whole ImageNet validation set (totally 50K images) (VQGAN: reconstruction images; Transformer: sampled images)? Should I split the dataset?

**Comment by rromb** [comment]
Thanks :) Reconstruction metrics should be evaluated on all 50k examples from the validation split. For transformer evaluation (i.e. sample quality), we follow standard practive and first generate 50k new samples and then evaluate against the full training set.
